lib/features/children/add_member/pages/add_member_counter_page.dart

```diff
@@ -11,9 +11,10 @@ import 'package:givt_app/shared/models/analytics_event.dart';
 import 'package:givt_app/shared/widgets/fun_scaffold.dart';
 
 class AddMemberCounterPage extends StatefulWidget {
-  const AddMemberCounterPage({this.initialAmount, this.showTopUp = false, super.key});
+  const AddMemberCounterPage({this.initialAmount, this.showTopUp = false, this.initialRegistration = true, super.key});
```

Comment:
I would not tie the naming to registration, canPop would be better for me.

Reply:
i did canPop + reversed the usage